scripts/policy/frameworks/cluster/agent/api.zeek

```diff
@@ -30,4 +30,11 @@ export {
 
 	# Report informational message.
 	global notify_log: event(instance: string, msg: string, node: string &default="");
+
+	# Notification events, controller -> agent
+
+	# Confirmation from controller in response to notify_agent_hello
+	# that the agent is welcome.
+	global notify_controller_hello: event(controller: string, host: addr);
```

Comment:
The same applies to the other events in this - and the other globals. Even though they were not changed in this PR, it would be neat to have them documented.